compiler/include/flags_list.h

```diff
@@ -118,6 +118,7 @@ symbolFlag( FLAG_FN_RETARG, npr, "fn returns via _retArg", ncm )
 symbolFlag( FLAG_FORMAL_TEMP,     npr, "formal temp", "a formal temp to back an in, out, or inout argument" )
 symbolFlag( FLAG_FORWARDING_FN , npr, "forwarding function" , ncm )
 symbolFlag( FLAG_FUNCTION_CLASS , npr, "function class" , "first-class function class representation" )
+symbolFlag( FLAG_FUNCTION_NEVER_RETURNS, ypr, "function never returns", "function that causes the program to exit" )
```

Comment:
Thanks!  I've renamed it to "function terminates program".  I'm doing another test run just to be on the safe side.